templates/branding/openqa.suse.de/sponsorbox.html.ep

```diff
@@ -0,0 +1,5 @@
+<div class="text-right" id="sponsorbox">
```

Comment:
I don't know what you mean. Should I not have any logo at all? Replace every reference of "sponsor" by "whatever" in openQA code or just use the openSUSE one?